tests/testthat/test-resample.R

```diff
@@ -201,7 +211,7 @@ test_that("can use `fit_resamples()` with a workflow - postprocessor (no trainin
       parsnip::linear_reg()
     ) %>%
     workflows::add_tailor(
-      tailor::tailor("regression") %>% tailor::adjust_numeric_range(lower_limit = 1)
+      tailor::tailor() %>% tailor::adjust_numeric_range(lower_limit = 1)
```

Comment:
I removed the deprecated `type` argument in tailor code I was editing and went ahead and did it in the last remaining couple places.